tests/utils.py

```diff
@@ -35,7 +36,9 @@ def setup_task(
             "binary_image": binary_image,
             "binary_image_resolved": binary_image + "-resolved",
             "bundle_mapping": {},
+            "deprecation_list": deprecation_list,
             "index_image": "feed.com/index/image:tag",
+            "index_image_resolved": "feed.com/index/image:tag",
```

Comment:
I don't see this related to this particular change. Could you please add this as a separate commit?

Also, it would be nice if you could squash your previous two commits into one.

Reply:
I will remove that.